lmsweb/models.py

```diff
@@ -174,6 +172,25 @@ def course_get_administrators(course_id):
     return course.get_course_administrators()
 
 
+ALL_MODELS = (User, Course, Lecture)
+
+from flask_admin import Admin
+from flask_admin.contrib.peewee import ModelView
+
+# set optional bootswatch theme
+app.config['FLASK_ADMIN_SWATCH'] = 'cerulean'
+
+admin = Admin(app, name='LMS', template_mode='bootstrap3')
+
+for m in ALL_MODELS:
+    admin.add_view(ModelView(m))
+
+
+def create_tables():
+    with database:
+        database.create_tables(ALL_MODELS)
+
+
 def test():
     database.drop_tables([User, Course, Lecture, UserCourseRelationship, UserLectureRelationship])
```

Comment:
99 chars in line

Reply:
Correct, although this is part of Sny's PR